packages/terra-clinical-result/tests/jest/ClinicalResult.test.jsx

```diff
@@ -62,6 +62,17 @@ describe('ClinicalResult', () => {
     expect(result).toMatchSnapshot();
   });
 
+  it('should render a result with no unit', () => {
+    const resultData = {
+      eventId: '111',
+      result: {
+        value: '12345.678',
+      },
+    };
+    const result = shallow(<ClinicalResult {...resultData} />);
+    expect(result).toMatchSnapshot();
+  });
```

Comment:
Can we add a test case where `result` is undefined?

Reply:
When result is undefined we don't render the visually hidden text, added a test for that here: [ffc5b1d](https://github.com/cerner/terra-clinical/pull/873/commits/ffc5b1db3c0cb80217f444379f2f2e7c5805ed56)